Flipkart-price-alert/README.md

```diff
@@ -0,0 +1,25 @@
+## Flipkart Price Alert
+Checks for the price of a desired product periodically.
+If the price drops below the desired amount, notifies the user via email.
+## Setup Instructions
+Type the following to install requirements: (make sure you have python installed)
+`pip install -r requirements.txt`
+# How to get app password
+1) To get app passwords, go to myaccount.google.com/security 
+2) Make sure you save 2-step verification turned on.
+3) Choose app passwords (under signing into google)
+4) Then under app choose mail and under platform, choose the device from which you are logged in from.
+
+![](https://devanswers.co/wp-content/uploads/2017/02/my-google-app-passwords.png)
+
+# Usage
+1) Edit the config vars in the track.py script.
+2) Run the track.py script. `python track.py`
+3) Hope the price drops :).
+
+## Author(s)
+Ea (github.com/jade9ja)
+
+## Disclaimers
```

Comment:
are you sure this disclaimer has anything to do with this script?

Reply:
The app passwords that one might generate for the script can still be used by others if the user accidentally leaks it hence the disclaimer for the same.

I will change/ remove the second disclaimer.